lib/model-helper.js

```diff
@@ -97,7 +97,7 @@ var modelHelper = module.exports = {
 
     out[name] = {
       id: name,
-      description: typeConverter.convertText(def.description),
+      description: typeConverter.convertText(def.settings.description),
```

Comment:
FWIW, this fails when `def.settings` is not defined, see existing unit-tests for an example.